compiler/objectfmt/OMRObjectFormat.hpp

```diff
@@ -124,7 +124,7 @@ class OMR_EXTENSIBLE ObjectFormat
     * @param[in] data : a populated \c TR::FunctionCallData structure with valid parameters
     *          for an encoded function call.
     */
-   virtual void printEncodedFunctionCall(TR::FILE *file, TR::FunctionCallData &data) = 0;
```

Comment:
@0xdaryl  While making changes on the Z,  I realized the print function for encoded function should return the bufferPos of the point after instruction encoded by it is printed as on Z this will be called during printing snippets for different calls and after the call instruction we will print the data put in the snippet.